### PR TITLE
refactor: move code snippets to files

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
It should help with simplifying the process to regenerate the plots. Not entirely sure what that looks like yet - ideally something fully automated but more likely just loading the files into a notebook. Either way, the code is more accessible since it's in `.py` files and broken up by plot.

### What the code is doing
Just moving things around, small fix to indentation, and running the formatting on the code.

### Testing
Built the docs locally and compared result with the [plotting section](https://breakthrough-energy.github.io/docs/postreise/index.html#plotting-scenario-data) currently deployed.

### Time estimate
10 min

### CODE REVIEW

File: docs/code/utilization_map.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from bokeh.io import show
+from powersimdata import Scenario
+
```
Comment (rouille) [inline docs/code/utilization_map.py:3]: It seems I forgot to declare `scenario`. Can you add `scenario = Scenario("3287"). Sorry
`

**Comment by rouille** [comment]
I imagine `black` and `flake8` went through the new python files

File: docs/code/utilization_map.py (hunk 1 above)
Comment (jon-hagg) [reply docs/code/utilization_map.py:3]: No problem - done

**Comment by jon-hagg** [comment]
> I imagine `black` and `flake8` went through the new python files

Actually looks like we only run `flake8` on the `postreise` folder otherwise it would have found the undefined `scenario` you caught

**Comment by rouille** [comment]
> > I imagine `black` and `flake8` went through the new python files
> 
> Actually looks like we only run `flake8` on the `postreise` folder otherwise it would have found the undefined `scenario` you caught

Yeah. Also, I noticed that there was only only one new line after the `import` statements

**Review by rouille — APPROVED** [review]
Looks good

**Pull request merged**